doc/requirements.txt

```diff
@@ -2,6 +2,7 @@
 pip
 appdirs
 autograd
+autoray
```

Comment:
Wait, and how comes a dependency was added to the docs?

Reply:
This is due to #1202 being merged in

Reply:
Due to the change above, this package, which is newly required due to #1202, needs to be added to `doc/requirements.txt` which #1202 could not have known or catch during tests because the change above is not merged in yet. Sorry for the mess :/
Separating the `doc/requirements.txt` issue into a separate PR would have been possible beforehand, but it would block the kernels module, which is why they are combined.

Reply:
Sorry for double commenting, I hadn't reloaded the page

Reply:
Ah, all good!